vaadin-text-field-flow-parent/vaadin-text-field-flow-integration-tests/src/test/java/com/vaadin/flow/component/textfield/tests/validation/TextAreaValidationBasicIT.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright 2000-2022 Vaadin Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.vaadin.flow.component.textfield.tests.validation;
+
+import com.vaadin.flow.component.textfield.testbench.TextAreaElement;
+import com.vaadin.flow.testutil.TestPath;
+import com.vaadin.tests.validation.AbstractValidationIT;
+import org.junit.Test;
+import org.openqa.selenium.Keys;
+
+import static com.vaadin.flow.component.textfield.tests.validation.TextFieldValidationBasicPage.*;
+
+@TestPath("vaadin-text-area/validation/basic")
+public class TextAreaValidationBasicIT
+        extends AbstractValidationIT<TextAreaElement> {
+    @Test
+    public void fieldIsInitiallyValid() {
+        assertClientValid();
+        assertServerValid();
+    }
+
+    @Test
+    public void onlyServerCanSetFieldToValid() {
+        $("button").id(REQUIRED_BUTTON).click();
+
+        executeScript("arguments[0].validate()", testField);
+        assertClientInvalid();
+
+        testField.sendKeys("Value");
+        executeScript("arguments[0].validate()", testField);
+        assertClientInvalid();
+
+        testField.sendKeys(Keys.TAB);
```

Comment:
Had to deviate here from the existing tests, because pressing enter in text area just adds a newline.